vanilla-framework/vanilla-framework
# 1947 slider pattern fix

## Done

Updated a variable in the pattern to use the interpolated syntax as suggested in [the issue](https://github.com/vanilla-framework/vanilla-framework/issues/1947)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check the [slider demo](http://0.0.0.0:8101/vanilla-framework/examples/patterns/slider/slider/) still works as expected

## Details

Fixes #1947
